src/convertor.rs

```diff
@@ -46,18 +50,22 @@ pub(crate) mod tests {
         next_day_starts_at: i64,
         timezone: Tz,
     ) -> Option<Vec<FSRSItem>> {
-        // Find the index of the first RevlogEntry in the last continuous group where review_kind = 0
-        // 寻找最后一组连续 review_kind = 0 的第一个 RevlogEntry 的索引
-        let mut index_to_keep = 0;
+        // Increment review_kind of all entries by 1
+        // 将所有 review_kind + 1
+        entries.iter_mut().for_each(|entry| entry.review_kind += 1);
```

Comment:
What do you think about the following alternative approach?

a) removing this
b) reducing the constants above by 1 each
c) adding a comment on the constants that they're one less than in the Python code, because we don't add 1 to everything

Reply:
It's OK because review_kind is used to filter out data only.